__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+const request = require("supertest");
+const app = require("../app");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+describe("GET /api/topics", () => {
+  test("status 200: should respond with an array of topic objects  ", () => {
```

Comment:
This is a really solid test, I love that you're testing your response is an array aswell as the length before you start looping to check properties to stop any false positives. 👍